test/gui/tst_addAccount/test.feature

```diff
@@ -46,3 +46,23 @@ Feature: adding accounts
         Then credentials wizard should be visible
 
 
+    Scenario: Adding account with virtual files advance configuration
+        Given the user has started the client
+        When the user adds the first account with advanced configuration
+            | server   | %local_server% |
+            | user     | Alice          |
+            | password | 1234           |
+        And the user selects virtual_files option in advanced section
+        And the user enables experimental feature
```

Comment:
```suggestion
        And the user selects enable_experimental_placeholder_mode option in enable experimental feature dialogue box
```
Just to make it consistent with the step that you have used in the scenario below this one.